spec/syntax.md

```diff
@@ -19,7 +19,7 @@
    1. [Preamble](#preamble)
    1. [Variants](#variants)
    1. [Patterns](#patterns)
-   1. [Placeables](#placeables)
+   1. [Placeholders](#placeholders)
```

Comment:
Might make sense for us to remove the index for now, to make editing easier. But that's a separate change from this PR.